carmin-server/server/common/exceptions.py

```diff
@@ -0,0 +1,3 @@
+class MissingRequiredParameterError(Exception):
```

Comment:
I would name this MissingRequiredParameterException instead of Error.

Reply:
[According to PEP8](https://www.python.org/dev/peps/pep-0008/#exception-names), this is the proper way of naming Exceptions.